src/commands.js

```diff
@@ -167,7 +167,6 @@ Commands.comfort = new Command("Comforts the person you want", (message, args) =
 Commands.say = new Command("Says whatever you want", (message, args) => {
     if (!args.join(" ")) { args[0] = "** **" }
     message.channel.send(args.join(" ").slice(0, 1900))
-    message.delete()
```

Comment:
why was this line deleted?

Reply:
If the first chain-command deletes the message, and the second one tries to do anything with the message, an error is thrown